llama_stack/providers/utils/inference/routable.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from typing import Dict, List
+
+from llama_models.sku_list import resolve_model
+
+from llama_stack.distribution.datatypes import RoutableProvider
+
+
+class RoutableProviderForModels(RoutableProvider):
```

Comment:
@yanxi0830 I updated the name to `validate_routing_keys()` instead and killed the `get_routing_keys()` method on this since it was confusing. Thanks for the feedback.